server/routers/matchTeam_router.js

```diff
@@ -28,44 +31,46 @@ exports.getRouter = function () {
   return router
 }
 
-function getDefaultMatchesForTeam (teamNumber) {
-  const practice = {
-    '_id': createRandomId(RANDOM_ID_LENGTH),
-    'matchId': 1,
-    'stage': 'practice',
-    'matchTeams': [
-      {
-        'teamNumber': teamNumber,
-        'tableId': null
-      }
-    ]
-  }
+function getDefaultMatchesForTeam (teamNumber, stages) {
+  const matches = []
 
-  const ranking = {
-    '_id': createRandomId(RANDOM_ID_LENGTH),
-    'matchId': 1,
-    'stage': 'ranking',
-    'matchTeams': [
-      {
+  for (const stage in stages) {
+    for (let i = 1; i <= stages[stage].matchAmount; i++) {
+      const match = {}
+      match._id = createRandomId(RANDOM_ID_LENGTH)
+      match.stage = stages[stage].stageName
+      match.matchId = i
+      match.matchTeams = [{
         'teamNumber': teamNumber,
         'tableId': null
-      }
-    ]
+      }]
+      matches.push(match)
+    }
   }
 
-  let matches = []
-  matches.push(practice)
+  return matches
+}
 
-  for (let i = 1; i <= 3; i++) {
-    let match = {}
-    match._id = createRandomId(RANDOM_ID_LENGTH)
-    match.matchId = i
-    match.stage = ranking.stage
-    match.matchTeams = ranking.matchTeams
-    matches.push(match)
-  }
+function getDefaultStages () {
+  const stages = [
+    {
+      stageName: 'practice',
+      matchAmount: 1
+    },
+    {
+      stageName: 'ranking',
+      matchAmount: 3
+    }
+  ]
 
-  return matches
+  let practice = settings.getSetting('practiceAmountOfMatches')
```

Comment:
Please use more indicative names. Here, and in the next line.

Reply:
Any idea for a better name?

Reply:
if you must, `practiceSettingsPromise`. I personally like to use these kind of promises inline in the call to `Promise.all`, this way they don't require a naming, which is always wierd with Promises.